pkg/driver/helmdriver.go

```diff
@@ -35,6 +35,10 @@ type helmDriver struct {
 
 var _ PackageDriver = (*helmDriver)(nil)
 
+// TODO: Temporarily hard coding but needs to be read from secrets
+var caFile = "/tmp/config/registry/CACERTCONTENT"
```

Comment:
update that to `/tmp/config/registry/ca.crt`

Reply:
Done